cli/loader_win_utils.c

```diff
@@ -17,7 +17,7 @@ int loader_fwrite(const char *str, size_t nchars, FILE *out) {
     if (out->isconsole) {
         // Windows consoles do not support UTF8, only UTF16.
         size_t wbufsz = nchars * 4 + 1;
-        wchar_t* wstr = (wchar_t*)loader_malloc(wbufsz);
+        wchar_t* wstr = (wchar_t*)malloc_s(wbufsz);
```

Comment:
There is no system `malloc` here (or even libc), just our `loader_malloc` (which already does not return NULL, akin to `malloc_s`)
```suggestion
        wchar_t* wstr = (wchar_t*)malloc(wbufsz);
```